cloud/passportauth/README.md

```diff
@@ -0,0 +1,75 @@
+# RainCatcher Passport Auth
+Raincatcher authentication and authorization module using [PassportJS][1]. This module creates and initializes a Passport authentication service using Passport's local strategy.
+This allows express routes to be protected from requests by checking if the user is authenticated and/or authorized with the required role. This module also
```

Comment:
'authenticated and/or authorized' needs better distinction, they're not synonims.

authenticated is just "I know you you are", authorized is "based on who you are, you are (not) allowed to access this"